lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -1262,6 +1262,14 @@ public void printSubpage(List<SimplePageItem> itemList, boolean first, UIBranchC
 			String color = null;
 			for (SimplePageItem i : itemList) {
 
+				//If the content MULTIMEDIA, type 7, is not released or is hidden will not be rendered on the view
+				if (i.getType() == SimplePageItem.MULTIMEDIA) {
+				    boolean isContentAvailable = contentHostingService.isAvailable(String.valueOf(i.getSakaiId()));
+				    if (!isContentAvailable) {
+				        continue;
+				    }
```

Comment:
One more small thing: Just remove the boolean variable assignment. It doesn't seem to be used anywhere else, so it's not needed.

if (!contentHostingService.isAvailable(String.valueOf(i.getSakaiId()))) {
    continue;
}